File: python/leetcode/math/number_theory/1217_chips.py
class Solution(object):
    def minCostToMoveChips(self, chips):
        """
        :type chips: List[int]
        :rtype: int
        """
        odd, even = 0, 0
        for item in chips:
            if item % 2 == 0:
                even += 1
            else:
                odd += 1
        return min(even, odd)

File: cc/leetcode/math/number_theory/1217_chips.cc
class Solution {
public:
    int minCostToMoveChips(vector<int>& chips) {
        int even = 0, odd = 0;
        for (int item : chips) {
            if (item % 2 == 1)
                odd++;
            else
                even++;
        }
        return min(odd, even);
    }
};
